TicTacToeApp/ComputerPlayer.swift

```diff
@@ -7,6 +7,16 @@ public class ComputerPlayer {
   }
 
   public func makeMove(gameState: Game) -> Int {
+    var move: Int?
+    let serviceUrl = "http://107.170.25.194:5000/"
+    let request = computerMoveRequest(gameState, baseUrl: serviceUrl)
+
+    makeBlockingRequest(request, onSuccess: { (data) in move = Int(data) })
+
+    return move ?? getRandomElement(availableSpaces(gameState))
+  }
+
+  private func availableSpaces(gameState: Game) -> [Int] {
```

Comment:
I think it would make sense for a `Board` to know it's own available spaces, however if you remove the `getRandomElement` branch, this method gets deleted anyway.